### ISSUE TITLE
Small typo

### ISSUE THREAD

**Issue by ghost** [Joystream/joystream#1032]
https://github.com/Joystream/joystream/blob/7ec3ac06a739f264353f9a531b1f37c9ed84d0ed/pioneer/packages/joy-proposals/src/Proposal/VotingSection.tsx#L101

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201703380142370/1201703511286477) by [Unito](https://www.unito.io)

**Comment by bedeho** [comment]
Thank you, would love a PR!

**Comment by ghost** [comment]
Yes, let this be my learning issue to figure GitHub out tomorrow between work, and apologies for overhead - I’ll be back with this 😊
-- 
 Securely sent with Tutanota. Get your own encrypted, ad-free mailbox: 
 https://tutanota.com


22 Jul 2020, 19:11 by notifications@github.com:

**Comment by bedeho** [comment]
No problem ;)
